checkpoint-multi-nic/custom-data.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
It reads an initialization script and encodes it into the custom data in azure deploy.json. 